about.css

```diff
@@ -0,0 +1,228 @@
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-size: 18px;
+    margin-bottom: 15px;
+    font-family: 'Rubik', sans-serif;
+}
+
+p {
+    line-height: 1.4;
+}
+
+.container {
+    width: 800px;
+    margin: 0 auto;
+}
+
+.container {
+    width: 800px;
+    margin: 0 auto;
+}
+
+header {
+    width: 800px;
+    display: flex;
+    justify-content: flex-end; 
+    margin-bottom: 30px;
+}
+
+header nav {
+    display: flex;
+    justify-content: space-between;
+    width: 610px; 
+    top: 10px;
+    margin-top: 40px;
+}
+
+header nav a {
+    text-decoration: none;
+    color: black;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: red;
+}
+
+.header-logo {
+    display: block;
+    position: absolute;
+    top: 15px;
+    left: 280px;
+}
+
+.top-image {
+    margin: 3px 5px 20px 5px;
+}
+
+.top-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed red;
+}
+
+.top-content .text-container {
+    width: 48%;
+    padding: 0 1%;
+    padding-bottom: 20px;  
+}
+
+.middle-content {
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+
+.middle-content h2 {
+    padding: 0 2%;
+    margin-bottom: 0;
+}
+
+.middle-content .boxes {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+
+.middle-content {
+    border-bottom: 5px solid black;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-items: center;
+	width: 95vw;
+	margin: 0 auto;
+}
+
+.middle-content .box1{
+   
+    width: 80px;
+    height: 80px;
+    border: 1px dashed red;
+    margin: 12px 20px 20px 40px;
+  }
+ 
+  .middle-content .box2 {
+    
+    width: 80px;
+    height: 80px;
+    border: 1px dashed red;
+    margin: 12px 20px 20px 40px;
+ }
+
+ .middle-content .box3 {
+    
+    width: 80px;
+    height: 80px;
+    border: 1px dashed black;
+    margin: 12px 20px 20px 40px;
+ }
+   
+ .middle-content .box4 {
+    
+    width: 80px;
+    height: 80px;
+    border: 1px dashed black;
+    margin: 12px 20px 20px 40px;
+ }
+ 
+
+ 
+ 
+ 
+.bottom-content {
```

Comment:
Check the margin and padding here I think its pushing the title text off a little bit.

Reply:
@Mykolb just checking in - did you make those changes?

Reply:
Hey! Sorry, I'm just seeing this. I made the changes. Turns out the css from the second page was overriding my code from the first page.

Reply:
@Mykolb nice!  definitely came together awesome stuff - there are a few styles on the buttons but otherwise its spot on - you can do the button stuff for practice during the week, otherwise MVP met and good job nice work on solving something I know that was frustrating you but you didn't give you and were able to get through it - nice work.